hbase-common/src/main/java/org/apache/hadoop/hbase/io/crypto/tls/X509Util.java

```diff
@@ -155,6 +154,8 @@ private static String[] getCBCCiphers() {
 
   private static final String[] DEFAULT_CIPHERS_OPENSSL = getOpenSslFilteredDefaultCiphers();
 
+  private static final Duration FILE_POLL_INTERVAL = Duration.ofMinutes(1);
```

Comment:
Why not configurable?

Reply:
It could be configurable, of course. I thought the potential value of that was low enough that it wasn't worth introducing a new config option.